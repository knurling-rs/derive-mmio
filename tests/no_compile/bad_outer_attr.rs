#[derive(derive_mmio::Mmio)]
#[mmio(no_ctors_x)]
#[repr(C)]
struct Uart {
    data: u32,
}

fn main() {}
