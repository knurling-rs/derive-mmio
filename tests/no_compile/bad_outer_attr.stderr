error: invalid content of mmio attribute, only expected `no_ctors`
 --> tests/no_compile/bad_outer_attr.rs:2:8
  |
2 | #[mmio(no_ctors_x)]
  |        ^^^^^^^^^^
